src/app.py

```diff
@@ -12,10 +12,13 @@
 from lib.yaraparse import parse_yara
 
 from util import mquery_version
-from db import Database, JobId, MQUERY_PLUGIN_NAME
+from db import Database, JobId
```

Comment:
We probably should think about import sorting :thinking:

Reply:
makes sense, we can do it together with reformat